moonbeam-types-bundle/index.ts

```diff
@@ -0,0 +1,143 @@
+import type { OverrideBundleDefinition, OverrideBundleType } from "@polkadot/types/types";
+export const moonbeamDefinitions = {
+  types: [
+    {
+      minmax: [0, 4],
+      types: {
+        AccountId: "EthereumAccountId",
+        Address: "AccountId",
+        Balance: "u128",
+        RefCount: "u8",
+        LookupSource: "AccountId",
+        Account: {
+          nonce: "U256",
+          balance: "u128",
+        },
+        ExitReason: {
```

Comment:
Doesn't `ExitReason` come with the Frontier stuff jaco added? If not, I think it should.

Reply:
Good catch, not sure why Jaco left them out https://github.com/polkadot-js/api/issues/3005

Reply:
ok he's fast ^^ https://github.com/polkadot-js/api/pull/3001/files